data/human/human missions.txt

```diff
@@ -7453,3 +7453,1280 @@ mission "Adelita 5"
+			`	Kormak frowns, "Lies.`
+			`	"Or, well this is more of folklore than a theory, that thousands of years ago humans were brought to the Deep by... elves."`
+			`	"Not elves," Kormak interrupts with a hint of annoyance,"the Dream Anuabhair. That how they called in my clan's sagas. The exceedingly proud folk. Never to trust."`
+			`	Kormak sighs and turns back to you, "If ever you stand before gleaming high-timbered hall built in shadow of dark Sidhe-Stone with carved face of Skoll-Wolf. Know that is my home."`
+			`	"Enough about past. Only present matters." He says and then goes back to eating.`
+			`	Tavish laughs, "Of course. Let us discourse about less weighty matters."`
+
+			label notathing
+			`	You talk a little of unimportant and unremarkable things beneath the glittering twilight stars in the midst of an untamed alien jungle.`
+			`	And then you head to bed.`
+			choice
+				`	(Sleep.)`
+
+			`	You are awoken by something warm and wet slowly sliding up your leg. Your tent has been perforated, and you can tell the blazing ring of fire is off.`
+			choice
+				`	(Kick whatever is on your leg.)`
+
+			`	With a few stiff kicks you force the grasping appendage to lets go of your leg and slithers back out the tent hole. You follow after it to see the entire campground is writhing in tenticle like vines.`
+			choice
+				`	(Shout for Tavish to do something!)`
+					goto tavshout
+				`	(Shout for Kormak to do something!)`
+					goto korshout
+				`	(Shout for everyone to do something!)`
+		
+			label tavshout
+			`	Tavish lunges out of his tent in a robe and screams, "The damned trees have returned!"`
+			`	He briefly kicks ineffectively at them before finding one that is trying to steal his machete and wrestling it free.`
+			`	"Kormak sometimes has a point," he grunts as he mashes back the various tendrils, "words don't work against monsters."`
+				goto everyone
+
+			label korshout
+			`	Kormak flies out of his tent completely naked. His rippling body outlined in dark blue tattoos. He rips and tears the vines with his bare hands and teeth. Eventually he finds his axe being dragged away and liberates it from the vine. Then he tears into the wild foilage with fearsome berserker rage. The trees scream in his bloody wake.`
+			`	"I am Gluniairn mac Kormakr!" He shouts into the night, "I am death!"`
+
+			label everyone
+			`	Everyone is now up and attacking the vines. Kormak is begins laughing and starts singing in his bloodlust:`
+			`	"Axe-Edge, Bright-Pledge, Breaker of Blight's Dredge,`
+			`	Ash-born, Eagle-torn, Slayer of Earth's forlorn.`
+			`	Break before my battle axe, Flee before my thunder.`
+			`	Die beneath a stranger's sky, and feed my clan's hunger."`
```

Comment:
```suggestion
			`	"Axe-Edge, Bright-Pledge, Breaker of Blight's Dredge,`
			`	"Ash-born, Eagle-torn, Slayer of Earth's forlorn.`
			`	"Break before my battle axe, Flee before my thunder.`
			`	"Die beneath a stranger's sky, and feed my clan's hunger."`
```

You could probably also just add line breaks and treat it like a poem (sort of like block quotes in an essay)

Reply:
Whatever makes it look best?

Reply:
Block-quote style would be:
```suggestion
			``
			`	Axe-Edge, Bright-Pledge, Breaker of Blight's Dredge,`
			`	Ash-born, Eagle-torn, Slayer of Earth's forlorn.`
			`	Break before my battle axe, Flee before my thunder.`
			`	Die beneath a stranger's sky, and feed my clan's hunger.`
			``
```